RealmSwift

```diff
@@ -0,0 +1 @@
+RealmSwift-swift1.2
```

Comment:
Yup, this needs to be here for Xcode to be able to load the config and thus get to ever running the switch script.